go.mod

```diff
@@ -7,4 +7,6 @@ require (
 	github.com/TexaProject/texajson v0.0.4
 	github.com/TexaProject/texalib v0.0.0-20170504075604-c488f5b05455
 	github.com/ethereum/go-ethereum v1.9.15
+	github.com/ipfs/go-cid v0.0.7
```

Comment:
Did you run `go mod clean` before making the push?

Reply:
no. Done now.